From 23f631387eed166eba68dd875e9a47b5e3e34bab Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 23 Nov 2018 20:59:25 +0900
Subject: [PATCH 521/909] arm64: dts: r8a77990-ebisu: Set preferred resolution

commit c39267cdbb3e91e25da76123d9b1651ffdeb7f53 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

On the Ebisu board, add option to set the initial resolution.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index 4854ab38cfff..10ddfb1ce8dc 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -170,6 +170,9 @@
 
 		no-use-ddc;
 
+		width = <1024>;
+		height = <768>;
+
 		port {
 			vga_in: endpoint {
 				remote-endpoint = <&adv7123_out>;
-- 
2.17.1

